src/Symfony/Bundle/FrameworkBundle/Command/CacheClearCommand.php

```diff
@@ -26,6 +29,42 @@
  */
 class CacheClearCommand extends ContainerAwareCommand
 {
+    private $filesystem;
+    private $cacheClearer;
+    private $cacheDir;
+
+    /**
+     * @param Filesystem            $filesystem
+     * @param CacheClearerInterface $cacheClearer
+     * @param string                $cacheDir
+     */
+    public function __construct($filesystem = null, CacheClearerInterface $cacheClearer = null, $cacheDir = null)
+    {
+        parent::__construct();
+
+        if (!$filesystem instanceof Filesystem) {
```

Comment:
I was going for that, but you still get different behavior :) before: exception, after: a path-ish command name.

A way good be to get the path from `getApplication()->getKernel()->getContainer()->getParameter()` =/